core/src/text/fontContext.cpp

```diff
@@ -57,6 +59,36 @@ FontContext::FontContext() :
             m_font[i]->addFace(m_alfons.addFontFace(alfons::InputSource(fontPath), size));
         }
     }
+#elif defined(PLATFORM_IOS)
```

Comment:
We definitely need a refactor of this once we have more knowledge of ios font loading, at least to get rid of the `PLATFORM_IOS` specific handling here.